transformer_engine/common/CMakeLists.txt

```diff
@@ -14,6 +14,12 @@ set(CMAKE_CUDA_STANDARD_REQUIRED ON)
 
 project(transformer_engine LANGUAGES CUDA CXX)
 
+if(NOT DEFINED NVTE_MAX_BUILD_THREADS)
+  set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --threads 4")
+else()
+  set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --threads ${NVTE_MAX_BUILD_THREADS}")
```

Comment:
I feel like users would get confused between `NVTE_MAX_BUILD_THREADS` and `NVTE_MAX_BUILD_JOBS`/`MAX_JOBS`. Maybe something like `NVTE_THREADS_PER_BUILD_JOB` is clearer.

Reply:
I would prefer to keep it as it is so that the two names are consistent. We can update docs.